tests/unit/igraph_get_shortest_paths_bellman_ford.c

```diff
@@ -0,0 +1,226 @@
+/* -*- mode: C -*-  */
+/*
+   IGraph library.
+   Copyright (C) 2006-2012  Gabor Csardi <csardi.gabor@gmail.com>
```

Comment:
This should now be `(C) 2006-2021 The igraph development team`, and the address in the next line is not necessary.